data/sql/updates/pending_db_world/rev_1611937625874584000.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611937625874584000');
+
+DELETE FROM `creature` WHERE `guid` = 15 AND `id` = 6491;
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(15, 6491, 571, 1, 1, 0, 0, 2002.7415771484375, 5938.0986328125, 37.18986129760742187, 5.934119224548339843, 300, 0, 0);
```

Comment:
lol 5.934119224548339843

5.93 or 5.934 would work as well

Reply:
Sniffed values my friend

Reply:
tell me if you see a difference with 5.934 IG

Reply:
sure, but what does exact position hurt us x)

Perhaps there won't be a noticeable difference ingame, but cool to have the correct values at least =)